src/main.sh

```diff
@@ -2,3 +2,4 @@
 # reset
 make
 ./cry.out
+make clean
```

Comment:
with `make clean`, the next `main.sh` invocation would re`make`, which would take quite some time :(